packages/mobile/src/screens/sign-on-screen/screens/CreateLoginDetailsScreen.tsx

```diff
@@ -0,0 +1,153 @@
+import { SignUpAgreementText } from '../components/SignUpPolicyText'
+import { Heading, Page, PageFooter, ReadOnlyField } from '../components/layout'
+import { Hint } from '../components/temp-harmony/Hint'
+import type { SignUpScreenParamList } from '../types'
+
+export type CreateLoginDetailsValues = {
+  email: string
+  password: string
+  confirmPassword: string
+}
+
+export const CreateLoginDetailsScreen = () => {
+  const dispatch = useDispatch()
+  const handleField = useSelector(getHandleField)
+  const existingEmailValue = useSelector(getEmailField)
+  const isVerified = useSelector(getIsVerified)
+  const navigation = useNavigation<SignUpScreenParamList>()
+  const { spacing } = useTheme()
+
+  const initialValues = {
+    email: existingEmailValue.value ?? '',
+    password: '',
+    confirmPassword: ''
+  }
+
+  const handleSubmit = useCallback(
+    (values: CreateLoginDetailsValues) => {
+      const { email, password } = values
+      dispatch(setValueField('email', email))
+      dispatch(setValueField('password', password))
+      navigation.navigate('FinishProfile')
+    },
+    [dispatch, navigation]
+  )
+
+  const audiusQueryContext = useAudiusQueryContext()
+
+  const loginDetailsFormikSchema = toFormikValidationSchema(
+    createLoginDetailsSchema(audiusQueryContext)
+  )
+
+  const navigateToLogin = () => {
+    navigation.navigate('SignOn', { screen: 'sign-in' })
+  }
+  return (
+    <Formik
+      initialValues={initialValues}
+      onSubmit={handleSubmit}
+      validationSchema={loginDetailsFormikSchema}
+    >
+      {({ isValid, dirty, errors }) => (
+        <Page>
+          <Heading
+            heading={messages.title}
+            description={messages.description}
+          />
+          <Flex direction='column' h='100%' gap='l'>
+            <Flex direction='column' gap='l'>
+              <ReadOnlyField
+                label={messages.handleLabel}
+                value={
+                  <Flex gap='xs'>
+                    <Text variant='body' size='m' color='default'>
+                      @{handleField.value}
+                      {isVerified ? (
+                        <IconVerified
+                          style={css({
+                            height: spacing.unit3,
+                            width: spacing.unit3
+                          })}
+                        />
+                      ) : null}
+                    </Text>
+                  </Flex>
+                }
+              />
+              <TextField label={messages.emailLabel} name='email' noGutter />
+              {errors.email === emailSchemaMessages.emailInUse ? (
+                <Hint icon={IconExclamationCircle}>
+                  <Text
+                    variant='body'
+                    size='m'
+                    style={css({ textAlign: 'center' })}
+                  >
+                    {emailSchemaMessages.emailInUse}{' '}
```

Comment:
oh interesting point that we might have a matching email at this point for log in... do we want that on this page you think?

Reply:
My thinking is to keep the behavior consistent and help avoid people making dupe accounts 🤷.
I can see a scenario where someone forgot how they signed up with Audius, clicks one of the social sign ups, then comes here and types in their email and sees that they already signed up before. (Personally I've been guilty of this exact scenario before lol)
If you're feeling otherwise, happy to discuss